<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoadRace</class>
 <widget class="QMainWindow" name="RoadRace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoadRace</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnStuff">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>115</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Test Case</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnOpenGame">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Open Game...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iptGameFileName">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playerName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green;
font-size:30px;</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Resources</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblWood">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wood</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stone</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblWa">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Water</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gold</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>731</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gameLayout" columnminimumwidth="0"/>
   </widget>
   <widget class="QLabel" name="wo1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Go1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Sto1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Wa1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name1">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.Helvetica Neue DeskInterface';&quot;&gt;Currently in single-player mode. Upgrade for more options.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpBtn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>85</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Help!</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="EsyBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Easy Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MedBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HardBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Hard Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cheatBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cheater...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>111</width>
      <height>27</height>
      <x>10</x>
      <y>390</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save the Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tst2Btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Use 7X7 test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Time (sec)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RanBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate a random World! </string>
     </property>
     </widget>
   <widget class="QPushButton" name="btnSendToServer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Send to Server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iptServerMsg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtServerOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="iptServerName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>localhost</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>640</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFrom_file">
   <property name="text">
    <string>From file...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
